apps/web/src/components/RangeInput.tsx

```diff
@@ -5,12 +5,16 @@ export type NumberRange = [bigint | undefined, bigint | undefined];
 export type RangeInputProps = {
   value: NumberRange;
   error?: string;
+  startPlaceholder?: string;
+  endPlaceholder?: string;
```

Comment:
they should be mandatory, right?

Reply:
The `placeholder` prop is usually optional in any input. We can have scenarios where we want to use a `label` instead of a `placeholder`